aep/general/0300/aep.md.j2

```diff
@@ -0,0 +1,98 @@
+# AEP Editions
+
+This AEP describes how the AEP specification is versioned.
+
+## Edition based versioning
+
+AEPs will be published in editions, similar but not identical to specifications
+such as those for the C++ or Rust programming language:
+
+- The specification will not change for a non-preview edition after it is
```

Comment:
Should we have a word for "non-preview"?  "Stable" maybe?

Reply:
sure, let's go with that.